src/App.test.jsx

```diff
@@ -20,17 +22,22 @@ const user = {
 }
 
 // 🚨 Create your server
-
+const server = setupServer(
+  rest.get(`${process.env.REACT_APP_SUPABASE_URL}/rest/v1/users`, (req, res, ctx) => res(ctx.json([user])))
+)
 // 🚨 Listen for server start
-beforeAll()
+beforeAll(() => server.listen());
+
+afterEach(() => server.resetHandlers());
 
 // 🚨 Close server when complete
-afterAll()
+afterAll(() => server.close())
```

Comment:
We will want to be able to access our Mock API call globally, which would require placing our beforeAll, afterAll, and server into the setupTests.js file.